README.rst

```diff
@@ -136,11 +136,23 @@ The official documentation is hosted on ReadTheDocs at http://larray.readthedocs
 Get in touch
 ============
 
-- Report bugs, suggest features or view the source code `on GitHub`_.
-- For questions, ideas or general discussion, use the `mailing list`_.
+- To be informed of each new release, please subscribe to this `mailing list`_.
+- For questions, ideas or general discussion, please go to our `Google Users Group`_.
+- To report bugs, suggest features or view the source code `on GitHub`_.
 
+.. _mailing list: https://groups.google.com/d/forum/larray-announce
+.. _Google Users Group: https://groups.google.com/d/forum/larray-users
 .. _on GitHub: http://github.com/liam2/larray
-.. _mailing list: https://groups.google.com/forum/#!forum/larray
+
+.. raw:: html
+
+   <p>
+   You can subscribe to the announce mailing list by entering your email address here:
```

Comment:
we probably want to add some explanation about delogging from any google account if one wants to use a non-gmail address.

Reply:
You mean to log out before to click on `Subscribe`?

Reply:
I **guess** so. Might want to test what is the easiest (before or after). But users who have a google account are otherwise invited to join with their gmail address.

Reply:
After some tests using http://larray-test.readthedocs.io/en/update_readme_174_500/, it seems that you must log out **before** to click on Subscribe.